aries-site/src/examples/components/tag/CreateTagSimple.js

```diff
@@ -0,0 +1,152 @@
+
+  return (
+    <AppContainer>
+      <TagPageHeader
+        description={`Tags are values that can be assigned to 
+      resources.`}
+      />
+      <Form
+        messages={{ required: 'This is a required field.' }}
+        onSubmit={() => {
+          const exists = currentTags.filter(t => t === value);
+          if (!exists.length) setCurrentTags([...currentTags, value]);
+          setValue(defaultValue);
+        }}
+      >
+        <Grid
+          columns={
+            size !== 'small'
+              ? [['auto', 'medium'], ['auto', 'medium'], 'auto']
+              : '100%'
+          }
+          gap={{ column: 'medium', row: 'small' }}
+        >
+          <FormField
+            name="tag-value-simple"
+            htmlFor="tag-value-simple"
+            label="Value"
+            required={{ indicator: false }}
+          >
+            <Select
+              name="tag-value-simple"
+              id="tag-value-simple"
+              placeholder="Select or create a value"
+              value={value}
+              options={valueOptions}
+              onChange={({ option }) => {
+                if (option.includes(prefix)) {
+                  defaultValueOptions.pop(); // remove Create option
+                  defaultValueOptions.push(valueSearch);
+                  setValueOptions(alphabetize(defaultValueOptions));
+                  setValue(valueSearch);
+                } else {
+                  setValue(option);
+                }
+              }}
+              onSearch={text => {
+                const nextValueOptions = defaultValueOptions;
+                updateCreateOption(text, nextValueOptions);
+                const exp = getRegExp(text);
+                setValueOptions(
+                  alphabetize(nextValueOptions.filter(o => exp.test(o))),
+                );
+                setValueSearch(text);
+              }}
+            />
+          </FormField>
+          <Box
+            justify="start"
+            // use theme style values to align Button with FormField input
+            pad={
+              size !== 'small'
+                ? {
+                    top: `${parseInt(theme.text.xsmall.height, 10) +
+                      parseInt(
+                        theme.global.edgeSize[theme.formField.label.margin.top],
+                        10,
+                      ) +
+                      parseInt(
+                        theme.global.edgeSize[
+                          theme.formField.content.margin.vertical
+                        ],
+                        10,
+                      )}px`,
+                  }
+                : undefined
+            }
+          >
+            <Button
+              alignSelf="start"
+              label="Assign"
+              type="submit"
+              fill={size === 'small' ? 'horizontal' : undefined}
+              secondary
+            />
+          </Box>
+        </Grid>
+      </Form>
+      <TagResults>
```

Comment:
Should "Tags to be assigned" read "Assigned Tags" or "Tags"?

Otherwise it seems like the example is incomplete. What do I do next?

Reply:
I think this confusion is coming from the fact that the design I drew from was showing these in a wizard context. There was a footer with buttons that indicated you were advancing to a new step and accepting your choices. For simplicity in the example, I hadn't included the footer buttons but I can see how the wording might be confusing. I will change the wording here to "Assigned Tags".